pages/index.html

```diff
@@ -4,9 +4,39 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../CSS/style.css">
+  <link rel="stylesheet" href="../CSS/index.css">
+  <title>Ice Cream Fansite</title>
 </head>
 <body>
-  
+  <div class="grid_container">
+    <header>
+      <h1 id="site-title">Ice Cream Fansite</h1>
+      <nav>
+        <a href="facts.html"><button class="btn">Fun Facts</button></a>
+        <a href="gallery.html"><button class="btn">Gallery</button></a>
+      </nav>
```

Comment:
Since this `nav` is inside the main `div` container, the text covers up the buttons when the screen is small (like on a laptop), making it impossible to click on the buttons.